src/component/TodoInput.js

```diff
@@ -0,0 +1,20 @@
+export default function TodoInput({ addTodo }) {
+  const $input = document.querySelector(".new-todo");
+
+  const handleAddTodo = ({ target, key }) => {
+    if (key !== "Enter") {
+      return;
+    }
+
+    const contents = target.value.trim();
+    if (contents === "") {
+      return;
+    }
```

Comment:
```suggestion
    if (key !== "Enter" || !target.value.trim()) {
      return;
    }
```

```false``` 조건이 많지 않아 이런 식으로 한번의 조건문으로 검사하는게 코드가 더 깔끔해질거 같습니다!

Reply:
개인적인 이벤트리스너 함수의 작성법이긴 합니다만...🤔

이벤트리스너 함수의 경우 두가지 부분으로 나눌 수 있다고 생각합니다.

- 이벤트를 target보다 상위에 등록했을 경우 올바른 target이 잡혔는지 검사하거나 key 이벤트의 경우에 keyCode를 확인하는 등, 이벤트의 실행 여부 자체를 검사하는 부분
- 실제로 동작하는 부분

이렇게요.

이 경우에 저는 contents를 추출하는 순간부터 실행 여부가 아닌 실제 실행부라고 판단해 이렇게 나눴는데.....
막상 이렇게 쓰다보니까 스스로도 좀 헷갈리는 부분이긴 하네요.🤔
조금 더 고민해보도록 하겠습니다! 😅

좋은 의견 감사해요 ㅎㅎ

Reply:
아하 저런 식으로 생각해서 사용할수도 있겠군요! hwookim님이 말씀해주신 의견도 충분히 설득력 있네요! 

새로운 관점을 하나 배워가네요! 감사합니다!!👍👍👍